app/Models/Forum/Topic.php

```diff
@@ -103,6 +103,12 @@ class Topic extends Model implements AfterCommit
         'topic_title' => 100,
     ];
 
+    const PLATFORM_ISSUE_TAGS = [
+        'lazer',
```

Comment:
`str_slug` the type (class name, translation, filename) and use the actual tag string everywhere else instead? The icon helper can just generate the full content (`stb` etc string for the new tags and `<i ...>` for everything else)